.github/workflows/prod-sync.yml

```diff
@@ -0,0 +1,44 @@
+name: Sync ODFE artifacts to Prod
+
+# For validation the client_payload can be empty
+# Make sure to pass the action as client_payload in the dispatch event for Prod Sync
+# Example: client_payload: { "action": "prod-sync"}
+
+on: 
+  repository_dispatch:
+    types: [prod-sync]
```

Comment:
prod-sync is quite confusing, considering using prod-sync-all

Reply:
made the change